listenbrainz/domain/spotify.py

```diff
@@ -30,6 +30,11 @@
     'playlist-modify-private',
```

Comment:
I originally added the playlist permissions to the listen permissions we request, but there isn't much point to doing anymore that if you're implementing them separately. AFAIR we only need them to export playlists to spotify from the front-end, which should be irrelevant now?
Or do we want to keep the ease of use aspect if a user connects their account they can export playlists manually without needing to go update their preferences?

Reply:
I think it makes sense to let playlist permissions to be a part of listening only for ease of use. If we separate those, we'll probably have to support multiple combinations as well. The separate variable I added for playlist perms is only to keep backend code clear.